tbx/static_src/sass/components/_listing-image.scss

```diff
@@ -16,6 +16,10 @@
         display: block;
     }
 
+    &__picture {
+        flex-shrink: 0;
```

Comment:
This fixes an issue in the 'more' content where the listing images were getting smaller than they should